apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -157,20 +157,22 @@ const parsedTitle = computedAsync(
 
 <template>
   <div class="w-full border-b last:border-b-0">
-    <button
-      class="w-full px-4 py-3 space-x-2 flex items-center justify-between"
-      @click="expanded = !expanded"
-    >
-      <div class="flex items-center overflow-hidden">
+    <div class="w-full px-4 py-3 gap-2 flex items-center">
+      <div class="flex items-center overflow-hidden gap-2 flex-auto">
```

Comment:
I think we can remove this extra `div` entirely so all left/button/right are at the same level.

Only difference I notice with and without is that with it left slot will be hidden on **super** small screens and without it it will stay.

Reply:
Extra div removed, and added a new one to prevent the "left" slot for being shrinked